mobile-workspace/latest/using/index.md

```diff
@@ -133,3 +133,51 @@ You can move files within the Mobile Workspace.
 
     * The scan will be saved where you navigated.
     * If you do not wish to keep the scan, exit without saving. A dialog box will appear to confirm you want to discard the scan, press **Discard**. Press **Cancel** to return to save your scan.
+
+## Tasks
+
+You can view, filter, complete, add comments, and add attachments to tasks while logged into the Mobile Workspace.
+
+### Tasks Overview
+
+The **Tasks** button is located at the bottom between **Favorites** and **Offline**. The Tasks list loads active tasks assigned to you or created by you. Task priority colors are set as the following: Low (green), Medium (orange), and High (red). Tasks will refresh when the user scrolls down to the end of the task list or when the refresh button is pressed. The refresh button will also reset filters.
+
+* A Dark mode for Tasks is set within user Settings under Appearance.
+* If a user is not able to view tasks, you will see this error:
+"This account is not configured on the process services."
+<!-- * If no tasks exist, you will see "Create a new task that you want to easily find later." -->
+
+### Task Filters
+
+Filters can be set for Status, Due date, and Task Name. Filter chips are shown at the top of the Task list and will show the active filter.
+
+1. Select a filter chip.
+
+2. Follow the directions for each filter as follows:
+
+    * A Status filter can be set by selecting “Active” or “Completed” and press **Apply**. The Status filter chip label will show the active filter, so “Completed” instead of “Status”.
+    * Press **Due date** to set a custom, date-ranged filter. “From” and “To” start and end fields set using a calendar date picker. Only one date needs to be set.
+    * Press **Task Name** to filter tasks by name. Type your search in a text field. This filter does not apply to priority and is not case sensitive.
+
+3. Press **Apply** to set the filter and **Reset** to reset the filter to default. The **X** in top right of the filter UI will close the window and return to the Task list.
+
+4. The filter chips will show the active filters and the refresh button will reset the filters.
+
+### Task view
+
+Within the Task view are details about the task: Task name, Due date, Priority, Assignee, Status, and Identifier. There is a back arrow to exit Task view and return to the Task list.
+
+Individual tasks have a “Comments” section below the task details. The most recent comment is shown with the user's icon, user name, date the comment was added, and the user’s comment. If a comment is too long, there is a **Read More** link. If there is more than one comment, you will see a link to **View all**. The link **Add a comment…** is below existing comments.
```

Comment:
There is no read more option on long text now.

Reply:
I've removed that sentence.